util/cgiwifi.c

```diff
@@ -161,9 +209,9 @@ CgiStatus ICACHE_FLASH_ATTR cgiWiFiScan(HttpdConnData *connData) {
 		connData->cgiData=(void *)1;
 		return HTTPD_CGI_MORE;
 	}
-#else
-	return HTTPD_CGI_DONE;
-#endif
+//#else
```

Comment:
These commented out and other commented out lines can also be removed. We don't want to carry around commented lines if there isn't any purpose for it.